src/eventHandlers/checkSuite/index.spec.ts

```diff
@@ -55,6 +65,7 @@ describe('check Suite event handler', (): void => {
                 edges: [],
               },
               state: 'OPEN',
+              title: 'bump @types/jest from 26.0.12 to 26.1.0',
```

Comment:
I don't think adding these `title` properties to the mock data is always necessary, but I erred on the side of consistency.